pom.xml

```diff
@@ -110,35 +110,40 @@
                 <!-- the Sling instance is provisioned from the model in src/main/provisioning/model.txt -->
                 <groupId>org.apache.sling</groupId>
                 <artifactId>slingstart-maven-plugin</artifactId>
-                <version>1.7.0</version>
```

Comment:
This version is usually managed. 1.7.16 should be managed in the latest parent pom.

Reply:
I agree that usually is the case. It is not in this case. In parent 30 it is 1.7.2 and this needs 1.7.16 - updating to the latest parent has other problems which I don't feel like addressing right now.